org.eclipse.wildwebdeveloper.xml/schema/XMLExtensions.exsd

```diff
@@ -0,0 +1,100 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.eclipse.wildwebdeveloper.xml" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="org.eclipse.wildwebdeveloper.xml" id="XMLExtensions" name="%ExtPoint.XMLExtensions"/>
+      </appinfo>
+      <documentation>
+         This extension point is used to contribute LSP4XML extension jars to extend the functionality of the XML language server.
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
+      <complexType>
+         <sequence>
+            <element ref="Jar" minOccurs="1" maxOccurs="unbounded"/>
+         </sequence>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+               <appinfo>
+                  <meta.attribute translatable="true"/>
+               </appinfo>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="Jar">
+      <complexType>
+         <attribute name="Path" type="string" use="required">
```

Comment:
lowercase `path`. Also type can be a "resource" to allow browsing the bundle content, get some validation et edition assistance in PDE editor.